src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/cloudconfigs/fdm/salesforce-cloud-service/_jcr_content/swaggerFile

```diff
@@ -0,0 +1,594 @@
+                    "type": "string"
+                },
+                "MailingGeocodeAccuracy": {
+                    "type": "string"
+                },
+                "MailingLatitude": {
+                    "type": "number"
+                },
+                "MailingLongitude": {
+                    "type": "number"
+                },
+                "MailingPostalCode": {
+                    "type": "string"
+                },
+                "MailingState": {
+                    "type": "string"
+                },
+                "MailingStreet": {
+                    "type": "string"
+                },
+                "MasterRecordId": {
+                    "type": "string"
+                },
+                "MobilePhone": {
+                    "type": "string"
+                },
+                "Name": {
+                    "type": "string"
+                },
+                "OtherAddress": {
+                    "type": "string"
+                },
+                "OtherCity": {
+                    "type": "string"
+                },
+                "OtherCountry": {
+                    "type": "string"
+                },
+                "OtherGeocodeAccuracy": {
+                    "type": "string"
+                },
+                "OtherLatitude": {
+                    "type": "number"
+                },
+                "OtherLongitude": {
+                    "type": "number"
+                },
+                "OtherPhone": {
+                    "type": "string"
+                },
+                "OtherPostalCode": {
+                    "type": "string"
+                },
+                "OtherState": {
+                    "type": "string"
+                },
+                "OtherStreet": {
+                    "type": "string"
+                },
+                "OwnerId": {
+                    "type": "string"
+                },
+                "Phone": {
+                    "type": "string"
+                },
+                "PhotoUrl": {
+                    "type": "string"
+                },
+                "ReportsToId": {
+                    "type": "string"
+                },
+                "Salutation": {
+                    "type": "string"
+                },
+                "SystemModstamp": {
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "Title": {
+                    "type": "string"
+                }
+            },
+            "required" : ["Phone", "Name", "Email"]
+        },
+		"SFResult": {
+			"type": "object",
+			"properties": {
+				"id": {
+					"type": "string"
+				}				
+			},
+			"required" : ["id"]
+		},		
+	  "All": {
+			"type": "object",
+			"properties": {
+			  "totalSize": {
+			    "type": "integer"
+			  },
+				"records": {
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/Records"
+					 }
+				}
+			}
+		},
+		"Records": {
```

Comment:
Hope you have discussed with PM for include all entities

Reply:
removed all entity and corresponding operation